toolchain/lower/file_context.h

```diff
@@ -157,6 +188,50 @@ class FileContext {
 
   auto BuildVtable(const SemIR::Class& class_info) -> llvm::GlobalVariable*;
 
+  // Function is used to track which specifics were lowered for each generic.
```

Comment:
[Function comment style](https://google.github.io/styleguide/cppguide.html#Function_Comments) is that comments should be written with an assumed "This function ...".

e.g. here, maybe:

```suggestion
  // Tracks which specifics were lowered for each generic.
```

I think you're generally phrasing comments a little differently from this guideline... Below, maybe it should be "Initializes" instead of "Initialize", and so on.

Reply:
I'm adding some more specific suggestions, but unresolving to help remind you on this for other functions.